### DISCUSSION TITLE
Decollate_batch() should be used with LoadImaged or LoadImage(image_only= "False") with dictionary_based input

### DISCUSSION THREAD

**Discussion by LaraSiblinii** [Project-MONAI/tutorials#1688] [General]
**Describe the bug**

Please only create issues here relating to the functioning of the tutorials. For all other questions/problems/bugs, please refer to MONAI's [main repository](https://github.com/Project-MONAI/MONAI). For MONAI bugs (not just a problem in the notebook), please create an [issue](https://github.com/Project-MONAI/MONAI/issues). For questions, please use the [Discussions tab](https://github.com/Project-MONAI/MONAI/discussions).

**To Reproduce**
2. Install '....'
3. Run commands '....'

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
 - OS
 - Python version
 - MONAI version [e.g. git commit hash]
 - CUDA/cuDNN version
 - GPU models and configuration

**Additional context**

**Comment by KumoLiu** [comment]
Hi @LaraSiblinii, could you please describe your issue more clearly?

convert to discussion for now.
Thanks.
